docs/extensions.md

```diff
@@ -25,7 +25,7 @@ The [pre-commit framework](https://pre-commit.com/) can run `semgrep` at commit-
 ```yaml
 repos:
 - repo: https://github.com/returntocorp/semgrep
-  rev: 'SEMGREP_VERSION_LATEST'
+  rev: 'v0.80.0'
```

Comment:
I'm guessing this change should not be included.

Reply:
yep, not sure where this came from :eyes: